app/page.tsx

```diff
@@ -20,17 +22,24 @@ const Home = () => {
         </Box>
       </Box>
       <Box
-        w={{ base: "base", xl: "xl" }}
-        h={{ base: "166px", md: "213px", xl: "261px" }}
+        w={{ base: "base", md: "741px", xl: "xl" }}
+        h={{ base: "323px", md: "411px", xl: "462px" }}
         p={{
-          base: "23px 23px 27px 23px",
+          base: "23px 24px 27px 24px",
           md: "37px 27px 28px 26px",
-          xl: "50px 127px 40px 127px",
+          xl: "50px 128px 40px 127px",
         }}
         m="auto"
       >
-        <Box h="100%" border="1px solid" borderColor="grey.400">
-          <Text textStyle="headerMedium">Cards and map box</Text>
+        <Box
+          h="100%"
+          border="1px solid"
+          borderColor="grey.400"
+          borderRadius="8px"
+          boxShadow="0px 2px 4px -1px #0000000F, 0px 4px 6px -1px #0000001A"
```

Comment:
Known TODO:
- [ ] need to pull this and other hardcoded values in via the theme file

Reply:
Moved this to a new issue #97